src/pantry.js

```diff
@@ -0,0 +1,52 @@
+import ingredientsData from '../src/data/ingredient-Data.js'
+
+class Pantry {
+  constructor(userID, pantryArray) {
+    this.id = userID
+    this.contents = pantryArray
+  };
+
+  determineIngredientsAvailable(recipe) {
+    return recipe.ingredients.every(recipeIng => {
```

Comment:
This whole class is impressive. I never thought about using .every for the recipe ingredients. Creative

Reply:
this .every method is a new one for me! Can't wait to learn all about it!